execution/executor-benchmark/src/transaction_generator.rs

```diff
@@ -440,49 +446,29 @@ impl TransactionGenerator {
         transactions_per_sender: usize,
         non_conflicting_txns: bool,
     ) {
```

Comment:
You need to make changes in the `run_workload` function instead to use the transaction generator.